### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by PrestonL**
Updated LICENSE file per conversation with project maintainer

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2014 Klaus Hartl
+Copyright 2006, 2014, 2015 Klaus Hartl & Fagner Brack
```
Comment (FagnerMartinsBrack) [inline LICENSE]: Why do we need to specify the dates like `2006`, `2014` and `2015`? Can't we just put 2018?

**Comment by FagnerMartinsBrack** [comment]
X-Ref: https://github.com/js-cookie/js-cookie/issues/424.

@PrestonL, given our conversation on https://github.com/js-cookie/js-cookie/issues/424, is there any other action to do in this regard?

**Comment by PrestonL** [comment]
I would recommend:

Copyright 2006-2018 Klaus Hartl, Fagner Brack, GitHub Contributors

or even with just the current year:

Copyright 2018 Klaus Hartl, Fagner Brack, GitHub Contributors

I believe the GitHub Terms of Service "Contributions Under Repository License" clause sufficiently covers everything and agree that no CLA is needed as discussed in #424 

The Terms of Service don't necessarily transfer copyright ownership of contributions to the project or maintainer, so attributing all the contributors as a whole does seem it would be an appropriate practice as OpenStreetMap does (they always say Copyright OpenStreetMap Contributors); has a side effect of gives good vibes by crediting the contributors as well.

Glad we were all able to dig into and learn something on that.

Thanks for all your work!

**Comment by FagnerMartinsBrack** [comment]
@PrestonL Do you wanna make the change then? After that change, it LGTM. Let's keep it for a while to see if there's any additional feedback.

Can you post a permalink to OpenStreetMap License or other examples for future context?

**Comment by PrestonL** [comment]
Sure, I created a new pull request and referenced this one in it.  Here is the example license and attribution format used by Open Street Map for reference:

https://www.openstreetmap.org/copyright

**Comment by FagnerMartinsBrack** [comment]
You don't need to create a new Pull Request, just push changes to the branch `patch-1` of your fork.

**Pull request closed without merging**